projects/js-packages/licensing/components/activation-screen-controls/style.scss

```
@@ -72,21 +72,32 @@
 	}
 }
 
-.jp-license-activation-screen-controls--button {
+.jp-license-activation-screen-controls--button,
+.jp-license-activation-screen-controls--button:active {
 	font-size: var(--font-body);
 
-	background-color: var(--jp-green-primary);
-	border-color: var(--jp-green-primary);
-	border-radius: var(--jp-border-radius);
-	color: var(--jp-white);
+	/* Black & White / Black */
+	background-color: #000000;
+	border-radius: 4px;
+
+	font-style: normal;
+	font-weight: 600;
+	font-size: 16px;
+	line-height: 24px;
+	/* identical to box height, or 150% */
+
+	/* Black & White / White */
+	color: #FFFFFF;
```

Do you think you could use the Jetpack references here, just in case we want to change blacks and whites everywhere at once in the future?

```suggestion
	background-color: var(--jp-black);
	border-radius: 4px;

	font-style: normal;
	font-weight: 600;
	font-size: 16px;
	line-height: 24px;
	/* identical to box height, or 150% */

	/* Black & White / White */
	color: var(--jp-white);
```